benchmarks/queries/clickbench/README.md

```diff
@@ -28,6 +28,26 @@ SELECT
 FROM hits;
 ```
 
+### Q1
```

Comment:
Filed https://github.com/apache/arrow-datafusion/pull/8950 to add these in their own PR